databags/download-tb-for-android-alpha.ini

```diff
@@ -0,0 +1,14 @@
+[Google Play]
+url = https://play.google.com/store/apps/details?id=org.torproject.torbrowser_alpha
+
+[arch64]
+url = https://dip.torproject.org/web/tpo/master/tor-browser-9.0a6-android-aarch64-multi.apk
```

Comment:
Ops, here you should replace with this URL:
https://dist.torproject.org/torbrowser/9.0a6/{tor-browser-VERSION-platform-ARCH.apk}

Or there's a macro to build this URL:
https://dip.torproject.org/web/tpo/blob/master/templates/macros/downloads.html

Reply:
Whoops! I thought there was something a little strange there. Guess I should've _checked_ the URLs. :laughing:

I'll look into using one of those macros.